sugoi-api-core/src/test/java/fr/insee/sugoi/core/service/UserServiceTest.java

```diff
@@ -230,4 +244,16 @@ public void getCertificateShouldFailWhenRealmNotFound() {
         RealmNotFoundException.class,
         () -> userService.getCertificate("idonotexist", "us2", "Toto"));
   }
+
+  @Test
+  @DisplayName(
+      "Given the default setting to verify the unicity of the mail is true and the real has no pre-defined configuration"
+          + " then we verify the mail")
```

Comment:
maybe end with something more explicit like : "we verify the mail does not already exist in the userstorage"